nomos-services/network/src/backends/libp2p.rs

```diff
@@ -23,6 +23,14 @@ pub struct Libp2p {
     commands_tx: mpsc::Sender<Command>,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct Libp2pInfo {
+    pub listen_addresses: Vec<Multiaddr>,
+    pub n_peers: usize,
+    pub n_connections: u32,
+    pub n_pending: u32,
```

Comment:
Nitpick: I know it may be pedantic but I think this would benefit of explicit names:
* peers_size
* connections_size
* pending_? (connections I guess)

Reply:
Lazy libp2p devs 😅
Ok, let's add connections to the last ones and that's it. Adding size was just and example though.